src/main/java/com/programmers/ticketparis/domain/reservation/Reservation.java

```diff
@@ -28,8 +33,16 @@ public class Reservation {
 
     @Builder
     private Reservation(ReservationStatus reservationStatus, Long customerId, Long scheduleId) {
-        this.reservationStatus = reservationStatus;
+        this.reservationStatus = checkReservationStatus(reservationStatus);
         this.customerId = customerId;
         this.scheduleId = scheduleId;
     }
+
+    private ReservationStatus checkReservationStatus(ReservationStatus reservationStatus) {
+        if (reservationStatus != ReservationStatus.COMPLETED) {
+            throw new ReservationException(NOT_EXIST_RESERVATION_STATUS, List.of(reservationStatus.toString()));
+        }
+
+        return reservationStatus;
+    }
```

Comment:
check 도 하고 return도 하네요. 하나만 하시죠 ㅎ

Reply:
```suggestion
    private void validateReservationStatus(ReservationStatus reservationStatus) {
        if (reservationStatus != ReservationStatus.COMPLETED) {
            throw new ReservationException(NOT_EXIST_RESERVATION_STATUS, List.of(reservationStatus.toString()));
        }
    }
```
한 메서드안에 2개의 의미를 담고 있는 것 같아서 반환 값은 void로 변경시키고 validate, 검증하는 동작만 하도록 변경했습니다!